jenkins/infra/jenkins/build-ci-image-all.groovy

```diff
@@ -134,6 +134,7 @@ pipeline {
 			script {
 				if (env.SHOULD_PUBLISH_JOB_STATUS?.toBoolean()) {
 					helper.sendDiscordNotification(
+						'CLIENT_DISCORD_WEB_HOOK_URL_ID',
```

Comment:
should build-ci notifications go to client or infra?

Reply:
yea, good catch 👍🏾